charts/kommander-operator/templates/pre_upgrade_kommandercore_hook.yaml

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
let's add a comment and create a ticket to remove this job in 2.8.0

Reply:
done